runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -4371,15 +4371,15 @@ MM_CopyForwardScheme::scanRoots(MM_EnvironmentVLHGC* env)
 
 			while (NULL != (classLoader = classLoaderIterator.nextSlot())) {
 				if (0 == (classLoader->gcFlags & J9_GC_CLASS_LOADER_DEAD)) {
-					if(J9_ARE_ANY_BITS_SET(classLoader->flags, J9CLASSLOADER_ANON_CLASS_LOADER)) {
+					if (J9_ARE_ANY_BITS_SET(classLoader->flags, J9CLASSLOADER_ANON_CLASS_LOADER)) {
```

Comment:
Since this code already uses `J9_ARE_ANY_BITS_SET()`, it should do so consistently (although I think the OMR equivalents should be preferred). The test here and similar ones on lines 1771,1772, 1786, 2909, 3269, etc., for example should be updated.